modules/highlight/highlight.go

```diff
@@ -184,7 +186,7 @@ func File(fileName, language string, code []byte) ([]string, string, error) {
 	lines := make([]string, 0, len(tokensLines))
 	for _, tokens := range tokensLines {
 		iterator = chroma.Literator(tokens...)
-		err = formatter.Format(htmlBuf, styles.GitHub, iterator)
+		err = formatter.Format(htmlBuf, githubStyles, iterator)
```

Comment:
Unrelated change?

Reply:
The golangci-lint bump made the linters fail on these lines (https://drone.gitea.io/go-gitea/gitea/69556/1/5) so I fixed them up too.

Reply:
I see. I wonder how this has even worked before, as I see no named export `GitHub` in that chroma module.

Reply:
> I see. I wonder how this has even worked before, as I see no named export `GitHub` in that chroma module.

https://github.com/alecthomas/chroma/blob/42c314dbe7e73e2ae4c853a93a05072e617e95a3/styles/compat.go#L28